modules/logging-clo-gui-install.adoc

```diff
@@ -0,0 +1,54 @@
+// Module is included in the following assemblies:
```

Comment:
@libander , @kalexand-rh pointed out to me something I missed in peer review. I think maybe you need to add some "+" to this procedure, so all the steps flow like 1,2,3,4,etc  instead of how they currently render:
![image](https://github.com/openshift/openshift-docs/assets/77501157/4d8dff43-d1c1-46ef-9c22-127d4a8a368c)